nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md

```diff
@@ -14,14 +14,15 @@ To handle any deserialization errors, wrap the `next()` operation in a try-catch
 
 snippet: DeserializationCustomization
 
+WARNING: Throwing the exception in the catch block will immediately forward the message to the error queue, if a message fails due to a `MessageDeserializationException` the message won't be retried. If that's not desired, remove the `throw` from the catch block to indicate that the message has been successfully processed.
 
 #### Handling Other Errors
 
 To handle any other errors, wrap the `next()` operation in a try-catch block and handle the `Exception` as shown:
 
 snippet: AllErrorsCustomization
 
-WARNING: Throwing the exception in the catch block will forward the message to the error queue. If that's not desired, remove the `throw` from the catch block to indicate that the message has been successfully processed.
+WARNING: Throwing the exception in the catch block will forward the message to the error queue after all the configured retry attempts, if the exception is not a `MessageDeserializationException`. If that's not desired, remove the `throw` from the catch block to indicate that the message has been successfully processed.
```

Comment:
```suggestion
WARNING: Throwing the exception in the catch block will forward the message to the error queue after all the configured retry attempts. If that's not desired, remove the `throw` from the catch block to indicate that the message has been successfully processed.
```

Reply:
Good change. It is not necessary to repeat the exception mentioning